sample/App/SampleApp.cpp

```diff
@@ -32,6 +33,7 @@ void SampleApp::Initialize()
     gameObject->SetName("TestObject");
 
     TestComponent* testComponent = gameObject->AddComponent<TestComponent>();
+    gore::Transform* transform   = gameObject->AddComponent<gore::Transform>();
```

Comment:
How about making Transform a default component (as in, automatically add it to new objects)?

Reply:
Ah you are right. I forgot to do this after a long journey with math libraries :w 
Wait for me